pkgs/applications/misc/cura/default.nix

```diff
@@ -52,7 +67,11 @@ mkDerivation rec {
 
   postFixup = ''
     wrapPythonPrograms
-    wrapQtApp $out/bin/cura
+
+    # find $out/bin
```

Comment:
Leftover debug code?

Reply:
Yes, I've only done the Cura dependencies, not Cura itself.